crates/fuel-core/src/graphql_api/storage.rs

```diff
@@ -123,6 +123,41 @@ pub enum Column {
     AssetsInfo = 25,
     /// Index of the coins that are available to spend.
     CoinsToSpend = 26,
+    /// See [`DaCompressionTemporalRegistryAddressV2`](da_compression::v2::DaCompressionTemporalRegistryAddressV2)
```

Comment:
So we have merkleized tables for `Address`, `AssetId`, `ContractId`, `ScriptCode` and `PredicateCode`. Why don't we need to merkleize the `RegistryIndex`, `Timestamps` and `EvictorCache` tables? Are those irrelevant during decompression?